### ISSUE TITLE
Missing Texture on Porto and No Trench Object Created

### ISSUE THREAD

**Issue by JHill29th** [gruppe-adler/grad_trenches#31]
```C#
17:30:02 "GRAD_Trenches: Type: ACE_envelope_big, Position: [0.475296,1.0276,-59.4225], WorldName: porto, SurfaceType: #D2Sand, Texture: ca\desert2\data\d2pisek_.paa"
17:30:02 Warning Message: Picture ca\desert2\data\d2pisek_.paa not found
```

After this script script error the interaction selection to build trench exists and the animation and progress bar appear but no trench object was produced.

**Comment by chris579** [comment]
Thanks for your bugreport. We will investigate the issue.

**Comment by chris579** [comment]
This issue is fixed in 1.4.1 which will go online on workshop tomorrow. However, it's already available on GitHub.
